features/react-native-fetch.feature

```diff
@@ -0,0 +1,200 @@
+    Then the exit code is successful
+    And the Content-Type header is valid multipart form-data
+    And the payload field "apiKey" equals "123"
+    And the payload field "appVersion" equals "2.0.0"
+    And the payload field "overwrite" equals "false"
+    And the payload field "dev" equals "<dev>"
+    And the payload field "platform" equals "<platform>"
+    # TODO(PLAT-5396) transforms are not applied so the source maps won't match
+    # And the payload field "sourceMap" matches the expected source map for "<TODO>"
+    And the payload field "bundle" matches the expected bundle for "<expected directory>"
+
+  Examples:
+    | platform | dev   | flags | version | expected directory                  |
+    | ios      | true  | --dev | 0-60    | fetch-react-native-0-60-ios-dev     |
+    | ios      | true  | --dev | 0-61    | fetch-react-native-0-61-ios-dev     |
+    | ios      | true  | --dev | 0-62    | fetch-react-native-0-62-ios-dev     |
+    | ios      | true  | --dev | 0-63    | fetch-react-native-0-63-ios-dev     |
+    | ios      | false |       | 0-60    | fetch-react-native-0-60-ios         |
+    | ios      | false |       | 0-61    | fetch-react-native-0-61-ios         |
+    | ios      | false |       | 0-62    | fetch-react-native-0-62-ios         |
+    | ios      | false |       | 0-63    | fetch-react-native-0-63-ios         |
+    | android  | true  | --dev | 0-60    | fetch-react-native-0-60-android-dev |
+    | android  | true  | --dev | 0-61    | fetch-react-native-0-61-android-dev |
+    | android  | true  | --dev | 0-62    | fetch-react-native-0-62-android-dev |
+    | android  | true  | --dev | 0-63    | fetch-react-native-0-63-android-dev |
+    | android  | false |       | 0-60    | fetch-react-native-0-60-android     |
+    | android  | false |       | 0-61    | fetch-react-native-0-61-android     |
+    | android  | false |       | 0-62    | fetch-react-native-0-62-android     |
+    | android  | false |       | 0-63    | fetch-react-native-0-63-android     |
+
+  Scenario: A request will be retried on a server failure (500 status code)
+    When I start the service "react-native-0-60-bundler"
+    And I wait for 2 seconds
+    And I set the HTTP status code for the next request to 500
+    And I run the service "react-native-0-60-fetch" with the command
+    """
+    bugsnag-source-maps upload-react-native
+      --api-key 123
+      --app-version 2.0.0
+      --endpoint http://maze-runner:9339
+      --fetch
+      --bundler-url http://react-native-0-60-bundler:9449
+      --platform ios
+    """
+    And I wait to receive 2 requests
+    Then the exit code is successful
+    And the Content-Type header is valid multipart form-data for all requests
+    And the payload field "apiKey" equals "123" for all requests
+    And the payload field "appVersion" equals "2.0.0" for all requests
+    And the payload field "overwrite" equals "false" for all requests
+    # TODO(PLAT-5396)
```

Comment:
I just closed PLAT-5369 as I realised I'd fixed it in #21.

Reply:
Awesome, I'll resolve these TODOs in this PR then